metricbeat/module/elasticsearch/index_recovery/_meta/data.json

```diff
@@ -1,38 +1,137 @@
 {
-    "@timestamp": "2017-10-12T08:05:34.853Z",
-    "agent": {
-        "hostname": "host.example.com",
-        "name": "host.example.com"
+    "@timestamp": "2020-08-26T20:34:41.034Z",
+    "@metadata": {
+        "beat": "metricbeat",
+        "type": "_doc",
+        "version": "8.0.0"
+    },
+    "service": {
+        "name": "elasticsearch",
+        "address": "localhost:9200",
+        "type": "elasticsearch"
+    },
+    "event": {
+        "dataset": "elasticsearch.index.recovery",
+        "module": "elasticsearch",
+        "duration": 30884428
+    },
+    "metricset": {
+        "name": "index_recovery",
+        "period": 10000
     },
     "elasticsearch": {
         "cluster": {
-            "id": "3LbUkLkURz--FR-YO0wLNA",
-            "name": "es1"
+            "name": "docker-cluster",
+            "id": "giRKn5pZQ0afY0YrRiXdxQ"
         },
         "index": {
-            "name": ".monitoring-es-6-2018.11.20",
```

Comment:
We are removing this field, which would be a breaking change. We should add it back.

Reply:
*Bump*

Reply:
Never mind, I see this has been updated now.